### ISSUE TITLE
UnicodeEncodeError: 'charmap' codec can't encode characters in position 28820-28821: character maps to <undefined>

### ISSUE THREAD

**Issue by voodoo709** [NocturnalNebula/litstash#3]
When running python litstash.py "https://www.literotica.com/s/the-assisted-wellness-initiative-ch-02" it generates an empty html file. Page two has emojis which I suspect is causing the issue.

I checked with Python 3.10.0 , 3.11.4, 3.12.0

```
python litstash.py "https://www.literotica.com/s/the-assisted-wellness-initiative-ch-02"
Detected 1 submission(s) in total

Downloading [Story] The Assisted Wellness Initiative Ch 02
[Saved] Page 1 of 4
[Saved] Page 2 of 4
[Saved] Page 3 of 4
[Saved] Page 4 of 4
Traceback (most recent call last):
  File "C:\litstash\litstash.py", line 1272, in <module>
    main()
  File "C:\litstash\litstash.py", line 1256, in main
    parseArgs(sys.argv[1:]) # send command line arguments to be parsed
  File "C:\litstash\litstash.py", line 1247, in parseArgs
    getList()
  File "C:\litstash\litstash.py", line 1073, in getList
    getSubmission(downloadList[0])
  File "C:\litstash\litstash.py", line 1056, in getSubmission
    obj.export() # export final string to html file
  File "C:\litstash\litstash.py", line 194, in export
    export(self.path, self.filename, self.output)
  File "C:\litstash\litstash.py", line 762, in export
    f.write(output)
  File "C:\Program Files\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 28820-28821: character maps to <undefined>




Python 3.11.4

C:\litstash>python litstash.py "https://www.literotica.com/s/the-assisted-wellness-initiative-ch-02"
Detected 1 submission(s) in total

Downloading [Story] The Assisted Wellness Initiative Ch 02
[Saved] Page 1 of 4
[Saved] Page 2 of 4
[Saved] Page 3 of 4
[Saved] Page 4 of 4
Traceback (most recent call last):
  File "C:\litstash\litstash.py", line 1272, in <module>
    main()
  File "C:\litstash\litstash.py", line 1256, in main
    parseArgs(sys.argv[1:]) # send command line arguments to be parsed
    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\litstash\litstash.py", line 1247, in parseArgs
    getList()
  File "C:\litstash\litstash.py", line 1073, in getList
    getSubmission(downloadList[0])
  File "C:\litstash\litstash.py", line 1056, in getSubmission
    obj.export() # export final string to html file
    ^^^^^^^^^^^^
  File "C:\litstash\litstash.py", line 194, in export
    export(self.path, self.filename, self.output)
  File "C:\litstash\litstash.py", line 762, in export
    f.write(output)
  File "C:\Program Files\Python311\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode characters in position 28820-28821: character maps to <undefined>






Python 3.12.0

C:\litstash>python litstash.py "https://www.literotica.com/s/the-assisted-wellness-initiative-ch-02"
Detected 1 submission(s) in total

Downloading [Story] The Assisted Wellness Initiative Ch 02
[Saved] Page 1 of 4
[Saved] Page 2 of 4
[Saved] Page 3 of 4
[Saved] Page 4 of 4
Traceback (most recent call last):
  File "C:\litstash\litstash.py", line 1272, in <module>
    main()
  File "C:\litstash\litstash.py", line 1256, in main
    parseArgs(sys.argv[1:]) # send command line arguments to be parsed
    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\litstash\litstash.py", line 1247, in parseArgs
    getList()
  File "C:\litstash\litstash.py", line 1073, in getList
    getSubmission(downloadList[0])
  File "C:\litstash\litstash.py", line 1056, in getSubmission
    obj.export() # export final string to html file
    ^^^^^^^^^^^^
  File "C:\litstash\litstash.py", line 194, in export
    export(self.path, self.filename, self.output)
  File "C:\litstash\litstash.py", line 762, in export
    f.write(output)
  File "C:\Program Files\Python312\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode characters in position 28820-28821: character maps to <undefined>
```

**Comment by voodoo709** [comment]
I've added a pull request to fix this error here https://github.com/NocturnalNebula/litstash/pull/4

**Comment by NocturnalNebula** [comment]
I was unable to reproduce this error on Python 3.11.6 on linux. The story downloaded with no problems. Anybody else?

**Comment by voodoo709** [comment]
I checked with Python 3.11.6 on Windows 11 with the latest 1.4 release and I’ve attached a screen capture video showing the error occurs with open(filename, 'w') and results in a 0kb file being saved whereas open(filename, 'w', encoding='utf-8') works and results in a 63kb file being saved.

Screen capture video: https://ufile.io/vo0evo1b

**Comment by NocturnalNebula** [comment]
Just released v1.5 and it should work now. Can you test it and let me know? As well as the story that resulted in a | in the filename?

Thanks
